### PR TITLE
Update Japanese translation (LINE 301-352)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @t-inu
/cc @atoato88 @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -298,58 +298,58 @@
...
-	"Go template format string for the config view output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd/config#ConfigViewTemplate": "",
-	"Go template format string for the status output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status": "",
+	"Go template format string for the cache list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate": "キャッシュ一覧出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status",
```
Comment (atoato88) [inline translations/ja.json:316]:
```suggestion
	"Go template format string for the cache list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate": "キャッシュ一覧出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#CacheListTemplate",
```
It's different from original URL for variables we can use.

File: translations/ja.json [hunk 2]
```diff
@@ -298,58 +298,58 @@
...
+	"Go template format string for the config view output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd/config#ConfigViewTemplate": "設定ビュー出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status",
```
Comment (atoato88) [inline translations/ja.json:317]:
```suggestion
	"Go template format string for the config view output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/\nFor the list of accessible variables for the template, see the struct values here: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd/config#ConfigViewTemplate": "設定ビュー出力用の Go テンプレートフォーマット文字列。Go テンプレートのフォーマットはこちら: https://golang.org/pkg/text/template/\nテンプレートでアクセス可能な変数の一覧は、こちらの構造化変数を参照してください: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd/config#ConfigViewTemplate",
```
Same comment above.

File: translations/ja.json [hunk 3]
```diff
@@ -298,58 +298,58 @@
...
+	"IP Address to use to expose ports (docker and podman driver only)": "ポートの expose に使用する IP アドレス (docker, podman ドライバーのみ)",
+	"IP address (ssh driver only)": "IP アドレス (SSH ドライバーのみ)",
+	"If present, writes to the provided file instead of stdout.": "指定すると、標準出力の代わりに指定されたファイルに出力します。",
+	"If set, automatically updates drivers to the latest version. Defaults to true.": "設定すると、自動的にドライバーを最新バージョンに更新します。デフォルトは true です。",
+	"If set, delete the current cluster if start fails and try again. Defaults to false.": "設定すると、現在のクラスターの起動に失敗した場合はクラスターを削除して再度試行します。デフォルトは false です。",
+	"If set, disables optimizations that are set for local Kubernetes. Including decreasing CoreDNS replicas from 2 to 1 and increasing kubeadm housekeeping-interval from 10s to 5m. Defaults to false.": "設定すると、最適化された設定をローカルの Kubernetes 用に無効化します。CoreDNS レプリカ数を 1 に減らし、kubeadm のハウスキーピング間隔を 10 秒から 5 分に増加させることを含みます。デフォルトは false です。",
+	"If set, download tarball of preloaded images if available to improve start time. Defaults to true.": "設定すると、開始時間を改善するため、利用可能であれば、プレロードイメージじの tar ボールをダウンロードします。デフォルトは false です。",
```
Comment (atoato88) [inline translations/ja.json:329]:
```suggestion
	"If set, download tarball of preloaded images if available to improve start time. Defaults to true.": "設定すると、開始時間を改善するため、利用可能であれば、プレロードイメージの tar ボールをダウンロードします。デフォルトは false です。",
```
Typo.

File: translations/ja.json [hunk 4]
```diff
@@ -298,58 +298,58 @@
...
+	"If set, pause all namespaces": "設定すると、全ネームスペースを一旦停止します",
+	"If set, unpause all namespaces": "設定すると、全ネームスペースを一旦停止解除します",
+	"If the above advice does not help, please let us know:": "上記アドバイスが参考にならない場合は、我々に教えてください:",
+	"If the host has a firewall:\n\t\t\n\t\t1. Allow a port through the firewall\n\t\t2. Specify \"--port=\u003cport_number\u003e\" for \"minikube mount\"": "ホストにファイアウォールがある場合:\n\t\t\n\t\t1. ファイアウォールを通過するポートを許可する\n\t\t2. 「minikube mount」用の「--port=\u003cポート番号\u003e」を指定する",
+	"If true, cache docker images for the current bootstrapper and load them into the machine. Always false with --driver=none.": "true の場合、現在のブートストラッパーの Docker イメージをキャッシュに保存して、マシンに読み込みます。--driver=none の場合は常に false です。",
+	"If true, cache docker images for the current bootstrapper and load them into the machine. Always false with --vm-driver=none.": "true の場合、現在のブートストラッパーの Docker イメージをキャッシュに保存して、マシンに読み込みます。--vm-driver=none の場合は常に false です。",
+	"If true, only download and cache files for later use - don't install or start anything.": "true の場合、後の使用のためのファイルのダウンロードとキャッシュ保存のみ行われます。インストールも起動も行いません",
+	"If true, pods might get deleted and restarted on addon enable": "true の場合、Pod は削除され、有効なアドオン上で再起動されます",
```
Comment (atoato88) [inline translations/ja.json:340]:
```suggestion
	"If true, pods might get deleted and restarted on addon enable": "true の場合、有効なアドオンのPod は削除され、再起動されます",
```

The place use this message at [here](https://github.com/kubernetes/minikube/blob/ecbaf8b3dc686a47a40e8d5abcd214e81d35149c/cmd/minikube/cmd/config/enable.go#L85)
Next, use this flag at [here](https://github.com/kubernetes/minikube/blob/ecbaf8b3dc686a47a40e8d5abcd214e81d35149c/pkg/addons/addons_gcpauth.go#L380).
[In detail](https://github.com/kubernetes/minikube/blob/ecbaf8b3dc686a47a40e8d5abcd214e81d35149c/pkg/addons/addons_gcpauth.go#L249), traget pods are deleted first, and created by same manifest.

So, I think it's better "Pod は削除され、再起動されます".

**Comment by atoato88** [comment]
I left some comments.
Especially, one comment dpends on logic implementation, so please recheck that.

File: translations/ja.json [hunk 5]
```diff
@@ -298,58 +298,58 @@
...
+	"IP Address to use to expose ports (docker and podman driver only)": "ポートの expose に使用する IP アドレス (docker, podman ドライバーのみ)",
+	"IP address (ssh driver only)": "IP アドレス (SSH ドライバーのみ)",
+	"If present, writes to the provided file instead of stdout.": "指定すると、標準出力の代わりに指定されたファイルに出力します。",
+	"If set, automatically updates drivers to the latest version. Defaults to true.": "設定すると、自動的にドライバーを最新バージョンに更新します。デフォルトは true です。",
+	"If set, delete the current cluster if start fails and try again. Defaults to false.": "設定すると、現在のクラスターの起動に失敗した場合はクラスターを削除して再度試行します。デフォルトは false です。",
+	"If set, disables optimizations that are set for local Kubernetes. Including decreasing CoreDNS replicas from 2 to 1 and increasing kubeadm housekeeping-interval from 10s to 5m. Defaults to false.": "設定すると、最適化された設定をローカルの Kubernetes 用に無効化します。CoreDNS レプリカ数を 1 に減らし、kubeadm のハウスキーピング間隔を 10 秒から 5 分に増加させることを含みます。デフォルトは false です。",
```
Comment (t-inu) [inline translations/ja.json:328]:
```suggestion
	"If set, disables optimizations that are set for local Kubernetes. Including decreasing CoreDNS replicas from 2 to 1 and increasing kubeadm housekeeping-interval from 10s to 5m. Defaults to false.": "設定すると、ローカルの Kubernetes 用に設定された最適化を無効化します。CoreDNS レプリカ数を 2 から 1 に減らし、kubeadm の housekeeping-interval を 10 秒から 5 分に増加させることを含みます。デフォルトは false です。",
```

- "ローカルの Kubernetes 用に最適化された設定を無効化" is better translation I think.
- Translation of "from 2" is missing.
- `housekeeping-interval` is an option parameter for kubeadm, so it's better to keep original text.

File: translations/ja.json [hunk 6]
```diff
@@ -298,58 +298,58 @@
...
+	"If set, download tarball of preloaded images if available to improve start time. Defaults to true.": "設定すると、開始時間を改善するため、利用可能であれば、プレロードイメージじの tar ボールをダウンロードします。デフォルトは false です。",
+	"If set, force the container runtime to use systemd as cgroup manager. Defaults to false.": "設定すると、cgroup マネージャーとして systemd を使うようコンテナーランタイムに強要します。デフォルトは false です。",
```
Comment (t-inu) [inline translations/ja.json:330]:
```suggestion
	"If set, force the container runtime to use systemd as cgroup manager. Defaults to false.": "設定すると、cgroup マネージャーとして systemd を使うようコンテナーランタイムに強制します。デフォルトは false です。",
```
I think "強制" is more commonly used.

File: translations/ja.json [hunk 7]
```diff
@@ -298,58 +298,58 @@
...
+	"If set, pause all namespaces": "設定すると、全ネームスペースを一旦停止します",
+	"If set, unpause all namespaces": "設定すると、全ネームスペースを一旦停止解除します",
+	"If the above advice does not help, please let us know:": "上記アドバイスが参考にならない場合は、我々に教えてください:",
+	"If the host has a firewall:\n\t\t\n\t\t1. Allow a port through the firewall\n\t\t2. Specify \"--port=\u003cport_number\u003e\" for \"minikube mount\"": "ホストにファイアウォールがある場合:\n\t\t\n\t\t1. ファイアウォールを通過するポートを許可する\n\t\t2. 「minikube mount」用の「--port=\u003cポート番号\u003e」を指定する",
+	"If true, cache docker images for the current bootstrapper and load them into the machine. Always false with --driver=none.": "true の場合、現在のブートストラッパーの Docker イメージをキャッシュに保存して、マシンに読み込みます。--driver=none の場合は常に false です。",
+	"If true, cache docker images for the current bootstrapper and load them into the machine. Always false with --vm-driver=none.": "true の場合、現在のブートストラッパーの Docker イメージをキャッシュに保存して、マシンに読み込みます。--vm-driver=none の場合は常に false です。",
+	"If true, only download and cache files for later use - don't install or start anything.": "true の場合、後の使用のためのファイルのダウンロードとキャッシュ保存のみ行われます。インストールも起動も行いません",
+	"If true, pods might get deleted and restarted on addon enable": "true の場合、有効なアドオンのPod は削除され、再起動されます",
```
Comment (t-inu) [inline translations/ja.json:340]:
```suggestion
	"If true, pods might get deleted and restarted on addon enable": "true の場合、有効なアドオンの Pod は削除され、再起動されます",
```
I missed this in my previous review. A blank is required.

**Comment by t-inu** [comment]
I've checked all the fixes.
/lgtm

**Review by s-kawamura-w664 — COMMENTED** [review]
There is no point from me.
LGTM too.

**Comment by yosshy** [comment]
@t-inu @s-kawamura-w664 @atoato88 thank you for your reviews.

/unhold
/assign @prezha
Japanese review finished. Please merge this.

**Review by spowelljr — APPROVED** [review]

**Pull request merged**